src/epics/revisions.js

```diff
@@ -19,11 +19,22 @@ const buildUrl = ( state ) => {
 	return url;
 };
 
-const fetchRevisions = ( action$, store ) => (
+export const hideLoader = ( action$ ) => (
+	action$
+		.ofType( 'PAGES_UPDATE' )
+		.flatMap( () => Observable.of( RevisionsActions.setStatusReady() ) )
+);
+
+export const showLoader = ( action$, store ) => (
 	action$
 		.filter( ( action ) => [ 'QUERY_UPDATE', 'QUERY_SET_VALUE', 'WIKIS_SET' ].includes( action.type ) )
-		// Ensure that all the necessary data is present.
 		.filter( () => !!store.getState().query.wiki && store.getState().wikis.size > 0 && store.getState().query.user.size > 0 )
+		.flatMap( () => Observable.of( RevisionsActions.setStatusFetching() ) )
+);
```

Comment:
@dayllanmaza 
Instead of having this as a separate action, it would be _better_ to use `Observable.concat()` within `fetchRevisions` to dispatch multiple actions (i.e. you can send the status fetching and another observable that will actually fetch the data).

Doing it separately is fine, but anytime one is updated, the other will need to be updated as well.

Reply:
Done